### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by mark-reijerkerk** [MustardChef/WSABuilds#180]
### Describe the bug | 描述一下这个错误

When installing the latest version without you installed it before there is an error message 
'Some files are missing in the folder'
After patching out the FileList check, it installs normal

### Steps to reproduce the issue | 重现该问题的步骤

Delete the all the files from the installation directory and from \WindowsSubsystemForAndroid_8wekyb3d8bbwe then install a blank version

### Expected behaviour | 预期的行为

Just install normally

### Downloaded Build Of WSA | 已下载的WSA版本

WSA_2308.40000.1.0_x64_Release-Nightly-with-Magisk-26.3-stable-MindTheGapps-13.0-as-Pixel-5-RemovedAmazon

### Windows build number | Windows构建号

23541

### PC Specification | 个人电脑规格

Does not apply

**Comment by MustardChef** [comment]
@mark-reijerkerk Are you using 7-zip to extract the files from the .7z archive?

**Comment by mark-reijerkerk** [comment]
Yes I do, but both Winrar and 7Z gives me a total of 167 files, while in the filelist there are 157 files

**Comment by MustardChef** [comment]
At what part of the installation process does the error occur, exactly? After WSA is installed and starts? Could you record the installation process and when the error occurs? (if its not too much of a hassle)

**Comment by mark-reijerkerk** [comment]
Line 88 of Install.ps1

**Comment by hunanflyer** [comment]
me too

**Comment by sadtango289** [comment]
Forgive me for the brief presentation because I'm not a professional > What is bug ? 
Please check screenshot https://i.imgur.com/KIrWOJN.png

**Comment by LeaderbotX400** [comment]
Still an issue on the latest version for me

**Comment by horhokokki** [comment]
Same here.

**Comment by ERR999R** [comment]
i just updated 7zip to the latest version, and unzipped it again. now its working successfully
